src/config/constants.js

```diff
@@ -1,7 +1,46 @@
 export const PUBLIC_IMAGE_FOLDER = '/images/';
+export const DEFAULT_BANNER_IMAGE = '/images/banners/default.png';
+export const banners = ['banners/cloud.jpg', 'banners/dns-server.png', 'banners/js.jpg', 'banners/full-stack-web-development.jpg', 'banners/load-balancer.png'];
+export const total = banners.length;
 
-export const DEFAULT_BANNER_IMAGE = 'banners/default.png';
+const selectOptions = [
+  {
+    label: 'Cricket',
+    value: 'cricket',
+  },
+  {
+    label: 'Football',
+    value: 'football',
+  },
+];
+const radioOptionsCricket = [
```

Comment:
cricketOptions is good enough. No need to mention select or radio in naming

Reply:
Changed Named to cricketOptions.